applications/app/views/fragments/indexBody.scala.html

```diff
@@ -58,6 +60,44 @@
         </div>
     }
 
+    case None if index.page.metadata.id == AudioFlagship.tagName => {
+
+        <div class="l-side-margins content--media" id="flagship-audio">
```

Comment:
I would prefer if this was in a separate file, like all the others. I know that is not the case for all the other cases but they're wrong 😆

Reply:
ah right... okay :)